tests/tst-pam_fail_delay.c

```diff
@@ -67,6 +67,8 @@ main (void)
       return 1;
     }
 
+  pam_end (pamh, retval | PAM_DATA_SILENT);
```

Comment:
Is there any particular reason to use `PAM_DATA_SILENT` in these added `pam_end()` invocations?